tchannel/glossary.py

```diff
@@ -18,10 +18,22 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+from __future__ import absolute_import
+
+import sys
+from . import __version__
+
 # Largest message ID supported by the system.
-#
 # Message ID 0xffffffff is reserved
 MAX_MESSAGE_ID = 0xfffffffe
 
 # CallRequestMessage uses it as the default TTL value for the message.
 DEFAULT_TIMEOUT = 30  # seconds
+
+TCHANNEL_LANGUAGE = 'python'
+
+# version format x.y.z
+TCHANNEL_LANGUAGE_VERSION = '.'.join(map(str, sys.version_info[0:3]))
+
+# version format x.y.z
+TCHANNEL_VERSION = __version__
```

Comment:
But we have protocol version and tchannel version. If I don't specially set it, it will be confusing.